api/src/main/java/org/openmrs/OrderSet.java

```diff
@@ -40,6 +40,8 @@
 	
 	private List<OrderSetMember> orderSetMembers;
 	
+	private Concept concept;
```

Comment:
Adding a category field means adding a field with name category.

Reply:
Yes I got that right, but when I should have the category field of class concept, that has got me hanging still a-bit.